Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -13,13 +13,13 @@ let notificationEventIDKey = "eventPlaceID"
 class EventNotificationsManager {
 
     // caches events by their start time by place
-    fileprivate lazy var sentNotifications: [String: Set<Date>] = {
-        var cache = [String: Set<Date>]()
-        guard let savedNotifications = UserDefaults.standard.dictionary(forKey: sentNotificationDictKey) as? [String: Set<Date>] else {
+    fileprivate lazy var sentNotifications: [String: [Double]] = {
```

Comment:
nit: What is this a mapping between? `sentNotifications` as a name doesn't tell me much.